### DISCUSSION TITLE
wHBTC lost when transferred from Solana  to ETH via Wormhole

### DISCUSSION THREAD

**Discussion by tosunra** [wormhole-foundation/wormhole#281] [Problems / bugs] [answered]
Hello , i transfer hbtc from Solana to Eth( wormhole) , i refresh the page, and the hbtc was sent, but i can not get them in my eth wallet. It just show In Process, how can i do now? Please help.

https://explorer.solana.com/tx/2TKPJtV7SGfAZdfXSuaa3zVMKNLo3Kjv1F1w87MBk4nHhQuRTZf7hVXFxSGkzxFYZtFdb76wSUk9KuARmBCFHrQk

**Comment by hendrikhofstadt** [comment, marked as answer]
You should be able to complete the transfer from the UI now. Sorry for the inconvenience. The bug has been fixed now. If not, please let me know and I'll help you complete the transfer.

**Reply by tosunra** [reply]
It is ok now. Thank you very much.
